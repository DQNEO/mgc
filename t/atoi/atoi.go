package main

import (
	"fmt"
	"strconv"
)

func f1() {
	var a string = "10485760"
	var i int
	i, _ = strconv.Atoi(a)
	fmt.Printf("%d\n", i-10485760) // 0

	a = "1"
	i, _ = strconv.Atoi(a)
	fmt.Printf("%d\n", i) // 1
}

func main() {
	f1()
}
