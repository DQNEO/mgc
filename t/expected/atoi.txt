0
1
0
7
10
100
1234567890
